packages/wellbeing-mock-data/wellbeing-mock-data.js

```diff
@@ -130,20 +130,106 @@ function createMockActivities() {
   }
 }
 
+
+function getRandomHomeButExcludeCurrent(current_id) {
+  var homeIds = Homes.find().map(function (e) { return e._id; });
+  // remove current home from array
+  var indexOfCurrent = homeIds.indexOf(current_id);
+  if (indexOfCurrent > -1) {
+    homeIds.splice(indexOfCurrent, 1);
+  }
+  //return random element from rest of the homes
+  return homeIds[Math.floor(Math.random() * homeIds.length)];
```

Comment:
Try using Lodash [_.sample](https://lodash.com/docs/4.17.4#sample) here to improve readability.

Reply:
By using `_.sample`, we gain a higher level, and succinct, tool for thinking. Now, we can  even sample multiple objects (using `_.sampleSize`) without much boilerplate or cognitive overhead. Using semantics like "sample" makes our code more closely reflect our goals (as opposed to the constraints of the machine interpreter). This improves readability, maintainability, and even the creativity (potential for creative expression) of the developer.